packages/xod-client/src/editor/components/inspectorWidgets/PulseTweakWidget.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PT from 'prop-types';
+import * as XP from 'xod-project';
+
+import PinWidget from './pinWidgets/PinWidget';
+
+class PulseTweakWidget extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.props.onSendTweakPulse(this.props.nodeId);
+  }
+
+  render() {
+    return (
+      <li>
+        <PinWidget
+          elementId="pulse-tweak"
+          label="OUT"
+          dataType={XP.PIN_TYPE.PULSE}
+          direction={XP.PIN_DIRECTION.OUTPUT}
+          isBindable
+          isConnected
+        >
+          <button
+            className="Button Button--light inspectorButton"
+            onClick={this.handleClick}
+          >
+            send pulse
```

Comment:
```suggestion
            Pulse
```

I mean “pulse” is a verb. Capitalized to be consistent with universal UI principles